src/pydp/algorithms/_algorithm.py

```diff
@@ -112,29 +106,17 @@ def quick_result(self, data: List[Union[int, float]]) -> Union[int, float]:
 
     def result(
         self,
-        privacy_budget: Union[float, None] = None,
```

Comment:
Can we not use a partial privacy budget anymore?

Reply:
We can't use partial privacy budget anymore. The idea is that the budget should be tracked on a higher level of DP system, not on the level of building block library.